packages/api-client/src/api/productDetail/product.ts

```diff
@@ -88,8 +88,8 @@ export default gql`
             mediaImages(search: { sort: ASC }) {
               ...MediaImages
             }
-            attributes(filter: $attributeFilters) {
-              ...Attribute
+            attributeValues(filter: { isPublic: true, page: $attributesPage, offset: $attributesOffset }) {
```

Comment:
leave `$attributeFilters` as they were before ( we don't want breaking changes if not needed)